pycloudlib/ibm/instance.py

```diff
@@ -746,9 +758,38 @@ def start(self, wait=True):
         if wait:
             self.wait()
 
+    def _check_instance_failed_status(self) -> None:
+        """
+        Check if the instance failed to start and raise an exception if so.
+
+        Raises:
+            IBMCapacityException: If the instance failed to start due to
+                capacity issues.
+            IBMException: If the instance failed to start due to other reasons.
+        """
+        if self._instance["status"] == _Status.FAILED.value:
+            if any(
+                "capacity" in reason["code"]
```

Comment:
Do we have a link pointing for docs covering this? If so, would you mind adding it for reference?

Reply:
I don't have any docs for this. I discovered this by manually launching and inspecting instance status via the API/SDK.